app/models/github_organization.rb

```diff
@@ -98,6 +98,20 @@ def team_invitations_url
     "https://github.com/orgs/#{login}/invitations/new"
   end
 
+  def create_organization_webhook(config: {}, options: {})
+    GitHub::Errors.with_error_handling do
+      hook_config = github_org_hook_default_config.merge(config)
+                                                  .tap { |hash| hash[:secret] = ENV['WEBHOOK_SECRET'] }
```

Comment:
Let's add this to the Rails secrets.yml file so that we don't pull from the `ENV`